hsmd/hsmd_wire.csv

```diff
@@ -23,6 +23,16 @@ msgdata,hsmd_init_reply,bip32,ext_key,
 msgdata,hsmd_init_reply,bolt12,point32,
 msgdata,hsmd_init_reply,onion_reply_secret,secret,
 
+# Declare a new channel.
+msgtype,hsmd_new_channel,30
+# Which identity to use for requests
```

Comment:
I don't see fields commented in other messages in this file, so maybe remove these comments for style consistency

Reply:
done